examples/prometheus/prometheus.yaml

```diff
@@ -291,3 +301,196 @@ objects:
         - source_labels: [__meta_kubernetes_service_name]
           action: replace
           target_label: kubernetes_name
+
+      alerting:
+        alertmanagers:
+        - scheme: http
+          static_configs:
+          - targets:
+            - "alertmanager:9093"
```

Comment:
I am not sure. should I use another oauth-proxy inside the alerts pod? or can the oauth-proxy support 2 apps? is there a simpler way?